manager/apis/app/v1alpha1/datacatalog_types.go

```diff
@@ -12,10 +12,10 @@ import (
 type DataStore struct {
 	// CredentialLocation is used to obtain
 	// the credentials from the credential management system - ex: vault
-	// +required
+	// +optional
 	CredentialLocation string `json:"credentialLocation"`
 	// Holds details for retrieving credentials by the modules from Vault store.
-	// +required
+	// +optional
```

Comment:
If a struct is optional it should be a pointer. 
=> ```Vault *Vault `json:"vault,omitempty"````

Reply:
Thanks, I fixed that.